src/client/hooks/useIsPath.ts

```diff
@@ -20,3 +21,8 @@ export const useIsPrint = () => ({
   print: useIsPath({ path: `${ClientRoutes.Assessment.Print.path.absolute}/*`, exact: false }),
   onlyTables: useIsPath({ path: `${ClientRoutes.Assessment.PrintTables.path.absolute}`, exact: true }),
 })
+
+export const useIsPanEuropean = () => {
+  const { assessmentName } = useParams<{ assessmentName: AssessmentName }>()
+  return assessmentName === 'paneuropean' || assessmentName === 'panEuropean'
```

Comment:
here better to use AssessmentNames.panEuropean instead of 'panEuropean'

Reply:
assessmentName should never be fully lowercase